### PR TITLE
feat/DEST-357/Adding circle ci integration

### PR DESCRIPTION
**Pull request by DhruvilSureshPatel**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+      - run:
+          name: Setup the environment for ansible-service
+          command: |
+            cd .deployment
+            ./ansible-setup.sh
+      - run:
+          name: Generate the hive properties and node properties file and push to cloud.
+          command: |
+            cd ../ansible-service
+            ansible-playbook -i environments/<< pipeline.parameters.environment>> trino.yml
```
Comment (91pratyush) [inline .circleci/config.yml:74]: Rename playbook to `trino-service` to keep it consistent with all names.

**Review by 91pratyush — CHANGES_REQUESTED** [review]

**Comment by 91pratyush** [comment]
@DhruvilSureshPatel add CodeOwners for this repo.

File: .circleci/config.yml (hunk 1 above)
Comment (hkarira1) [reply .circleci/config.yml]: changed the playbook name @91pratyush

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+          command: pip3 install jinja-cli
+      - run:
+          name: Render task definition json using jinja for trino service
+          command: |
+            export image_tag=$(echo "$CIRCLE_BRANCH" | sed -e "s/\//_/g")
+            export ENVIRONMENT=<< pipeline.parameters.environment >>
+            export CONTAINER_IMAGE="475116478827.dkr.ecr.${AWS_REGION}.amazonaws.com/trino-service:${image_tag}"
+            export MODULE="trino-service"
+            export TRINO_HIVE_CONFIG_PATH="${TRINO_HIVE_CONFIG_PATH}"
+            jinja -E ENVIRONMENT -E ECS_TASK_ROLE_ARN -E ECS_EXECUTION_ROLE_ARN -E CONTAINER_IMAGE -E MODULE -E AWS_REGION -E CIRCLE_SHA1 -E CORALOGIX_API_KEY -E FIRELENS_FLUENTBIT_CONFIG_BUCKET .deployment/task-definition.json.j2 -o .deployment/task-definition.json
```
Comment (vighneshhevo) [inline .circleci/config.yml:111]: what will CIRCLE_SHA1 hold?

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+  GHA_Meta:
+    type: string
+    default: ""
+  environment:
+    type: string
+    default: "test"
+  ansible-branch:
+    description: Ansible branch to use for configs
+    type: string
+    default: "master"
```
Comment (vighneshhevo) [inline .circleci/config.yml:25]: can we change the default branch to dev as we have given default environment to test?

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+      - run:
+          name: Install jinja cli
+          command: pip3 install jinja-cli
+      - run:
+          name: Render task definition json using jinja for trino service
+          command: |
+            export image_tag=$(echo "$CIRCLE_BRANCH" | sed -e "s/\//_/g")
+            export ENVIRONMENT=<< pipeline.parameters.environment >>
+            export CONTAINER_IMAGE="475116478827.dkr.ecr.${AWS_REGION}.amazonaws.com/trino-service:${image_tag}"
+            export MODULE="trino-service"
```
Comment (vighneshhevo) [inline .circleci/config.yml:109]: Nit: can we replace `module` with `application`?

**Review by tremendoustj — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (hkarira1) [reply .circleci/config.yml]: I have removed this as this is not required.

**Review by 91pratyush — APPROVED** [review]

File: .circleci/config.yml (hunk 3 above)
Comment (hkarira1) [reply .circleci/config.yml]: changed this to dev everywhere.

**Review by 91pratyush — APPROVED** [review]

**Review by tremendoustj — APPROVED** [review]

**Pull request merged**